openquake/hazardlib/gsim/bchydro_epistemic.py

```diff
@@ -0,0 +1,450 @@
+# -*- coding: utf-8 -*-
```

Comment:
The name of this file does not comply with the usual standard we used. Are you planning to change it once the GMPE will be in its final form?

Reply:
I had not planned to because this is a set of adjustments to an existing GMPE and not a new model with a specific publication in itself. But I should change to `bchydro_2016_epistemic.py` to clarify this applies to the 2016 BC Hydro model and not subsequent BC Hydro models.